### ISSUE TITLE
Department Of State: Application:  Education and Transcript

### ISSUE THREAD

**Issue by Lforeman22** [openopps/openopps-platform#2486]
Who: Student
What: Add Education and Transcripts to application
Why: The student is required to provide education information and a transcript to their application.

Acceptance Criteria:
- All sections of this card are required. 
- There will be 3 questions with a Yes and No radio button
  - Are you currently enrolled (part-time or full-time) or accepted for enrollment in an accredited college or university?
  - Will you be, at a minimum, a college or university junior (i.e. have completed 60 or more undergraduate semester credit hours or 90 or more undergraduate quarter credit hours) by the start of the intern session for which you are applying?
  - Will you continue your education after this internship has been completed?
- There will be 1 question with a free form box
  - What is your cumulative GPA n a 4.0 scale?
- Degree information will be pulled from the USAJOBS shared profile. 
- Each degree card will have the following information:
  - The type of degree (Master's Degree, Bachelor's Degree)
  - The name of the school in bold black
  - The date of the degree 
- Clicking the Degree Name Link will expand the card. 
- Clicking Add Education Link will take you to the Education and Transcript page to add additional information. 
- There will be a header titled Transcript 
- A list of transcripts stored on USAJOBS will be displayed
- The user will select the transcript by clicking the radio button
- When the user clicks "upload transcript" they will be taken to USAJOBS to upload a transcript into their USAJOBS profile
- There will be a “What if I don’t have a transcript?” Link which will take the user to  the help center page. 
- Clicking Save and Continue will bring you back to the Application Process. 
- Clicking the back button will take you back to the Experience and Reference Page. 
- Clicking Save and Continue will take you to the Foreign Languages & Skills Step of the Application Process. 

InVision Mock: https://opm.invisionapp.com/d/main/#/console/15360465/319289334/preview

**Comment by Lforeman22** [comment]
Waiting on feedback from Tracy once she speaks to Security about how to handle this process.

**Comment by Piglady64** [comment]
Corresponding TFS Ticket:https://tfs.opm.gov/tfs/RecruitmentSystems/USAJOBS/_queries?id=35516&fullScreen=false&_a=edit

**Comment by kbwagner** [comment]
Duplicate of 2879 - closing
